addons/l10n_at/data/account_tax_report_data.xml

```diff
@@ -60,8 +68,15 @@
             <record id="tax_report_line_l10n_at_tva_sale_report_title" model="account.report.line">
                 <field name="name">4. VAT Computation (U1/U30)</field>
                 <field name="name@de">4. Berechnung der Umsatzsteuer (U1/U30)</field>
-                <field name="aggregation_formula">(AT_022_tax.balance + AT_029_tax.balance + AT_006_tax.balance + AT_037_tax.balance + AT_052_tax.balance + AT_007_tax.balance + AT_056.balance + AT_057.balance + AT_048.balance + AT_044.balance + AT_032.balance + AT_072_tax.balance + AT_073_tax.balance + AT_008_tax.balance + AT_088_tax.balance)</field>
+                <field name="code">AT_0004</field>
                 <field name="hierarchy_level">0</field>
+                <field name="expression_ids">
+                    <record id="tax_report_line_l10n_at_tva_vat_total" model="account.report.expression">
```

Comment:
```suggestion
                    <record id="tax_report_line_l10n_at_tva_total" model="account.report.expression">
```
"tva" and "vat" are the same thing, one in French and the other in English, so no need to have both :wink: 
I would usually suggest to use the English term, but since the French "tva" is used everywhere else, let's stick to that for the sake of consistency :nerd_face: